### DISCUSSION TITLE
GraphQL server connection to postgreSQL

### DISCUSSION THREAD

**Discussion by bovealexandre** [nuxt/nuxt#21954] [Questions]
Hi,
does someone already tried to connect his postgresql database to a graphql server with the likes of knex? Everything I found didn't work

**Comment by bovealexandre** [comment]
someone already did the integration of postgraphile to nuxt?

**Reply by Hebilicious** [reply]
Yes, have a look at the discord, I've recently someone talking about using nuxt and postgraphile together.

**Reply by bovealexandre** [reply]
ok everthing works but not thanks to the discord :p 

so right now I have my generated schema but when I try to import a certain query it doesn't work it transforms my query like graphql.macro so I don't understant how to select my specific query

so it looks like

```
definitions: Array(44) [ {…}, {…}, {…}, … ]
​
kind: "Document"
​
loc: Object { start: 0, end: 26188, source: {…} }
```
